app/views/records/edit_fields/_nested_geo_location_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<div id="generic-file-location-form" class="form-group">
+  <div class="geo_point nested-field repeating-field">
+    <fieldset>
+      <ul class="listing">
+        <h4>Point</h4>
+        <%= f.simple_fields_for :nested_geo_points do |a| %>
+          <li class="field-wrapper">
+            <% unless a.object.new_record? %>
+              <%= a.input :id, :as => :hidden %>
+            <% end %>
+
+            <%= a.input :label, :wrapper_html => {:class => "col-xs-12"} %>
+            <%= a.input :latitude, :wrapper_html => {:class => "col-xs-6"} %>
+            <%= a.input :longitude, :wrapper_html => {:class => "col-xs-6"} %>
+          </li>
+        <% end %>
+      </ul>
+    </fieldset>
+  </div>
+
+  <div class="geo_bbox nested-field repeating-field">
+    <fieldset>
+      <ul class="listing">
+        <h4>Bounding box</h4>
+        <%= f.simple_fields_for :nested_geo_bbox do |a| %>
+          <li class="field-wrapper">
+            <% unless a.object.new_record? %>
+              <%= a.input :id, :as => :hidden %>
+            <% end %>
+
+            <%= a.input :label, :wrapper_html => {:class => "col-xs-12"} %>
+            <%= a.input :bbox, :wrapper_html => {:class => "col-xs-12 bbox_field"} %>
+            <%= a.input :bbox_lat_north, :wrapper_html => {:class => "col-xs-6"} %>
```

Comment:
bbox_lat_north, bbox_lat_west, bbox_lat_south, and bbox_lat_east could use nice labels for the form, you can override the form label like so;

`<%= a.input :bbox_lat_north, :label => "Latitude North", :wrapper_html => {:class => "col-xs-6"} %>`